eluthia/zipapp_script.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+from apps import config
+import subprocess
+import zipfile
+import os
+import time
+
+
+class InstallationError(Exception): # Used to signify that installation / verification failed and we should roll back
+    pass
+
+
+def get_version(package, deb_file=False):
```

Comment:
I prefer you have a separate function to get the version of the deb file, where the first argument would be the path to the deb file. eg. `def get_version_of_deb_file(path):`